src/coreclr/tools/aot/ILCompiler.ReadyToRun/IBC/MIbcProfileParser.cs

```diff
@@ -35,7 +35,7 @@ TypeSystemEntityOrUnknown IPgoSchemaDataLoader<TypeSystemEntityOrUnknown>.TypeFr
                 try
                 {
                     if (token == 0)
-                        return new TypeSystemEntityOrUnknown(0);
+                        return new TypeSystemEntityOrUnknown(null);
```

Comment:
@davidwrighton does this change look fine? Otherwise `TypeSystemEntityOrUnknown.IsNull` returns false for these.

Reply:
Yes, this looks good.